boutdata/squashoutput.py

```diff
@@ -22,7 +22,8 @@
 def squashoutput(datadir=".", outputname="BOUT.dmp.nc", format="NETCDF4", tind=None,
                  xind=None, yind=None, zind=None, xguards=True, yguards="include_upper",
                  singleprecision=False, compress=False, least_significant_digit=None,
-                 quiet=False, complevel=None, append=False, delete=False):
+                 quiet=False, complevel=None, append=False, delete=False,
+                 tind_auto=False):
```

Comment:
Would it not make more sense to default to `True` given the new error thrown in `boutdata/data.py`?

Reply:
No - `tind_auto` (maybe not the best name, but I'm copying the argument that was already there in `collect()`) is only useful if there was an error writing the output in the first place - it's a work-around. `tind` is more useful (for general slicing in the t-dimension) and can only be used when `tind_auto` is `False`.